### PR TITLE
first draft civic file

### PR DESCRIPTION
**Pull request by tomBeach**
First version of civic.json file

### CODE REVIEW

File: civic.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+{
+    "id": "",
```
Comment (alulsh) [inline civic.json]: Should be a URL to the schema specification per [the DC gov website](http://open.dc.gov/civic.json/specification.html).

We can use

`"id": "https://raw.githubusercontent.com/DCgov/civic.json/master/schemas/schema-v1.json",`

from https://github.com/codefordc/civic.json/blob/master/schema.json#L3

File: civic.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            "url": "http://codefordc.org",
+            "name": "Code for DC"
+        },
+        {
+            "url": "http://www.waba.org/",
+            "name": "Washington Area Bicyclist Association"
+        }
+    ],
+    "description": "Creating a bike map advocacy tool for Washington Area Bicyclist Association",
+    "license": "",
```
Comment (alulsh) [inline civic.json]: License is a *should* be included field [per the specification](http://open.dc.gov/civic.json/specification.html), let's remove from the `civic.json` until https://github.com/dcfemtech/hackforgood-waba-map/issues/52 is closed.

Once we decide on a license we should then add it to the `civic.json`.

File: civic.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            "name": "buffers",
+            "url": "https://github.com/dcfemtech/hackforgood-waba-map/tree/master/buffers"
+        },
+        {
+            "name": "bikelanes",
+            "url": "https://github.com/dcfemtech/hackforgood-waba-map/tree/master/bikelanes"
+        }
+    ],
+    "tags": [
```
Comment (alulsh) [inline civic.json]: We should add `DC` and `WABA` as tags here to make this project easier to search for.

File: civic.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        "Data visualization",
+        "mapping",
+        "biking",
+        "bike lanes"
+    ],
+    "links": []
```
Comment (alulsh) [inline civic.json]: Similar to the license property, let's remove `links` if blank. If we want to keep then I'd add links to the project demo and the WABA website.

File: civic.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (alulsh) [inline civic.json]: Remove extra line of whitespace - first line should be `{`.

**Comment by alulsh** [comment]
Thanks for taking a stab at this @tomBeach. I added some line notes for minor things. Once those are tackled I can merge this.

**Comment by alulsh** [comment]
@tomBeach - any update on this? This is good work and I'd love to merge it soon! If not, I'm happy to tackle the line comments myself as well.

**Comment by alulsh** [comment]
Whoops this is now saying it's from an `unknown repository` - I'm going to use the trick in https://speakerdeck.com/holman/more-git-and-github-secrets?slide=114 to fetch this and push as a feature branch to this repo so we can merge.

**Comment by tomBeach** [comment]
Sortof forgot about this PR but have now pushed a modified version of this civic.json file with the changes mentioned on new branch civicjson

**Comment by alulsh** [comment]
@tomBeach - I did a compare for your new `civicjson` branch in https://github.com/dcfemtech/hackforgood-waba-map/compare/civicjson.

Looks like this includes the new Mapbox GL JS commits from @hrecht's WIP branch in https://github.com/dcfemtech/hackforgood-waba-map/pull/97.

Mind creating a new branch that doesn't have the Mapbox GL JS changes and only modifies the `civic.json` file? This will make your civic.json branch considerably easier to merge. We will likely need to modify @hrecht's code in her PR - it's a Work In Progress (WIP).

**Comment by alulsh** [comment]
Closing in favor of https://github.com/dcfemtech/hackforgood-waba-map/pull/128.

**Pull request closed without merging**